clients/lcdproc/disk.c

```diff
@@ -30,6 +31,28 @@
 #include "disk.h"
 #include "util.h"
 
+#define DISK_IGNORE_MAX 10
+
+/**
+ * Checks if a mountpoint is on the "ignore" list. 
+ * 
+ * \param mount			The mountpoint being evaluated
+ * \param ignore_list	List of ignore entries from the configuration.
+ * \return true if mount is present in ignore_list[], false otherwise.
+ */
+bool 
+disk_is_ignored(const char* mount, char* ignore_list[DISK_IGNORE_MAX])
```

Comment:
As this is a local helper, I'd make this static instead of (default) extern.

Reply:
To confirm - `char* x` vs `char *x`?

Reply:
No.

static bool disk_is_ignored(...) - then the symbol is local to the file (the compiler will do optimizations - probably just inline everything since it's used in one place only) otherwise the symbol is added to the dynamic linking table, which adds overhead and (very theoretically) risks collisions with symbols in other files or even libraries.

Reply:
Oops, I replied to the wrong comment with that question; I was referring to the "spacing" change request. Sorry.